docs/working-drafts/gwde-dfdl-experience-9-experimental-number-reps.adoc

```diff
@@ -0,0 +1,88 @@
+:title: Extended Binary Number and Binary Calendar Representations
+:title1: DFDL Experimental Feature
+:author: Mike Beckerle
+:authorOrg: (Apache Daffodil)
+:email: mbeckerle@apache.org
+:toc: macro
+:toc-title:
+[frame=none, grid=none]
+|===
+<|GWD-E >|{author} {authorOrg}
+<|OGF DFDL WG
+|
+<|dfdl-wg@ogf.org >|November 2022
+|===
+''''''''''''''''''''''''''
+[discrete]
+= {title1}
+[discrete]
+= {title}
+''''''''''''''''''''''''''
+[discrete]
+===== Status of This Document
+Grid Working Document (GWD)
+[discrete]
+===== Copyright Notice
+Copyright (C) Open Grid Forum, 2022. Some Rights Reserved. Distribution is Unlimited.
+[discrete]
+===== Abstract
+This document provides experience information to the OGF community on the Data Format Description Language (DFDL) 1.0 specification (GFD-P-R.207)
+
+It documents experimental features which add new binary number representations and binary calendar representations.
+[discrete]
+===== Contents
+toc::[]
```

Comment:
In the PDF the TOC does not contain the sub-sections of the back-matter. It needs to.